meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2023.01.inc

```diff
@@ -26,7 +26,7 @@ SRC_URI:append = " file://target_env/"
 SRC_URI:append = " file://btrfs.cfg"
 SRC_URI:append = " file://0001-REVERTME-usb-dwc3-gadget-skip-endpoints-ep-18-in-out.patch"
 
-PV = "2022.10"
+PV = "2023.01"
```

Comment:
Somewhere you mentioned 2023.10, is it a typo here?

Reply:
Same here _2301.01.inc, is automatic in PV. So it is actually correct, but this file gets renamed in later patch. I left the history, just so that in future we can revert to 2023.01 if needed.